### DISCUSSION TITLE
Little question

### DISCUSSION THREAD

**Discussion by ZenFlora** [ppy/osu#17876] [Q&A] [answered]
Will lazer be prioritized in a few years and stable will no longer be updated?
I hope stable gets updated often, even if lazer is prioritized. It would be sad if stable was no longer updated and players that can't afford to update pc get stuck on the old version of osu. I personally can't play lazer since I am on windows 8, but I'll be upgrading soon.

If this was already answered, sorry for wasting your time!

**Comment by smoogipoo** [comment, marked as answer]
Lazer is already our focus and osu-stable no longer receives major updates.

**Reply by ZenFlora** [reply]
Oh, okay thanks! What would stable updates be about?

**Reply by smoogipoo** [reply]
Critical fixes only, see https://github.com/ppy/osu-stable-issues

**Reply by ZenFlora** [reply]
Thank you!
